### ISSUE TITLE
Create a stack of 64 itens from 1 item or less than 64.

### ISSUE THREAD

**Issue by braiansaia** [tom5454/Toms-Storage#311]
I'm playing the Proeminence II (RPG) and that modpack has your Toms Storage.

I find a really annoying bug, that is ruining my experience. I have made a room full of chests and im using the Toms Storage mod. Its working normally, but when i remove a item, that have less than 64 it creates a full stack of that itens. Look at the pictures.
![Captura de tela 2024-01-05 225447](https://github.com/tom5454/Toms-Storage/assets/49910446/a185dfa2-20ff-47e0-9e92-9cf6845be211)
![Captura de tela 2024-01-05 225508](https://github.com/tom5454/Toms-Storage/assets/49910446/d7e3d602-a06a-4947-9a38-92b2ceaa0071)

**Comment by tom5454** [comment]
Do you have any drawer like blocks attached? Are you using vanilla chests, or some other modded chests?

**Comment by braiansaia** [comment]
Yes. We fixed that storaged, it was bugged when it was close to Create machines, making it to generate infinite itens.

**Comment by b0ps** [comment]
We have the same issue on our Server, and we also have Create mod vaults connected to the system as well as drawers from Extended Drawers.  Are there any fixes known yet?

**Comment by b0ps** [comment]
> We have the same issue on our Server, and we also have Create mod vaults connected to the system as well as drawers from Extended Drawers. Are there any fixes known yet?

Mod Version: 1.20-1.6.8 Fabric
